src/crispy/flags.h

```diff
@@ -150,7 +150,7 @@ struct fmt::formatter<crispy::flags<Enum>>: public fmt::formatter<std::string>
                 continue;
 
             if (!result.empty())
-                result += '|';
+                result += ',';
```

Comment:
was there a reason why you did it or had to do this? If you rely on that in the YAML output, than we should not use the fmt formatter here to encode for the YAML output. this is formatting only for human readability. :)

Reply:
This is not needed anymore, we wrap format inside yaml writer